apps/user-office-backend/src/queries/QuestionaryQueries.ts

```diff
@@ -127,6 +127,13 @@ export default class QuestionaryQueries {
     return this.dataSource.getBlankQuestionarySteps(templateId);
   }
 
+  async getBlankQuestionaryStepsByCallId(
+    agent: UserWithRole | null,
+    callId: number
+  ): Promise<QuestionaryStep[]> {
+    return this.dataSource.getBlankQuestionaryStepsByCallId(callId);
```

Comment:
suggestion: Maybe you could fetch the template id in this resolver and  then just reuse the flow already in place
```
 async getBlankQuestionaryStepsByCallId(
    agent: UserWithRole | null,
    callId: number
  ): Promise<QuestionaryStep[]> {
    const call = await this.callDataSource.getCall(callId);
    if (!call) {
      logger.logWarn('Could not get blank questionary steps by call id', {
        email: agent?.email,
        callId,
      });

      return [];
    }
`